app/build.gradle

```diff
@@ -3,16 +3,22 @@ plugins {
     id 'kotlin-android'
     id 'kotlin-android-extensions'
 }
+//test
 
 apply plugin: 'kotlin-kapt'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'org.jetbrains.kotlin.android.extensions'
+androidExtensions {
+    experimental = true
+}
 
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.3"
 
     defaultConfig {
         applicationId "com.example.pangea"
-        minSdkVersion 21
+        minSdkVersion 19
```

Comment:
Why do we need to downgrade the version from 21 to 19?

Reply:
sorry i forgot to revert that. I needed to use minSdkVersion 19 so that i could connect Android Studio with my tablet.